### PR TITLE
work on dmx keypad

### PR DESCRIPTION
**Pull request by daveol**
TODO:
- [x] Make keypad design with buttons
- [x] Make input buttons work
- [x] Process the input on enter

This also includes a small css fix for mobile, the navigation tabs in the universe views don't stack anymore if they don't fit now it scrolls horizontally if there is not enough room to fit them next to each other

### CODE REVIEW

**Comment by nomis52** [comment]
LGTM

File: javascript/new-src/src/js/app.js [hunk 1]
```diff
@@ -63,9 +63,8 @@ angular
         var integers = [];
         for (var i = OLA.MAX_CHANNEL_NUMBER; i >= OLA.MIN_CHANNEL_NUMBER; i--) {
           var value = channelValueCheck(dmx[i - 1]);
-          if (value > OLA.MIN_CHANNEL_VALUE ||
-              !strip ||
-              i === OLA.MIN_CHANNEL_NUMBER){
+          if (value > OLA.MIN_CHANNEL_VALUE || !strip ||
+            i === OLA.MIN_CHANNEL_NUMBER) {
```
Comment (peternewman) [inline javascript/new-src/src/js/app.js]: Can we indent to after the ( above?

**Comment by peternewman** [comment]
LGTM too

**Comment by daveol** [comment]
I think it's done now. keyboard input comes later since that is a extra request...

**Comment by nomis52** [comment]
Wonderful! I've tested it on a mobile as well and it all works.

**Comment by peternewman** [comment]
LGTM

**Pull request merged**